src/libcore/num/mod.rs

```diff
@@ -2037,6 +2038,120 @@ assert_eq!((-a).rem_euclid(-b), 1);
             }
         }
 
+        doc_comment! {
+            concat!("Returns the logarithm of the number with respect to an arbitrary base.
+
+Returns `None` if the number is smaller than 1.
+
+The result may not be optimized owing to implementation details;
```

Comment:
I think the method would be optimized, not the result which would be the same :)

Reply:
Good point; changed the phrasing (: